README.md

```diff
@@ -1,10 +1,10 @@
 # <p align = "center"> Web3.o.World </p>
 
```

Comment:
Where is the cover image ? Please include that also by seeing in the original Readme.md